src/components/TokenDisplay.tsx

```diff
@@ -68,7 +70,15 @@ function TokenDisplay({
 
   const amountToShow = showLocalAmount ? amountInLocalCurrency : new BigNumber(amount)
 
-  const sign = hideSign ? '' : amountToShow.isNegative() ? '-' : showExplicitPositiveSign ? '+' : ''
+  const sign = hideSign
```

Comment:
I think we should keep the approx symbol and +/- signs separate. This doesn't seem like the right abstraction, and it creates dependencies between the two options (e.g., if `hideSign` and `showApprox` are both true, `hideSign` takes precedence).